minesweeper.py

```diff
@@ -266,33 +273,28 @@ def flagging(event):
     "The force is strong with this one.",
 ]
 
-
-def highscore(mins, secs):
-    converted_to_seconds = mins * 60 + secs
-    for time in top_10_times:
-        if converted_to_seconds < time and len(top_10_times) < 10:
-            top_10_times.insert(
-                index, converted_to_seconds
-            )  # HOW DO I GET THE INDEX OF "TIME" IN FOR LOOP?
-        elif converted_to_seconds < time and len(top_10_times >= 10):
-            top_10_times.remove(top_10_times[-1])
-            top_10_times.insert(index, converted_to_seconds)  # SAME AS ABOVE
-
-
+def highscore():
+    data = where_this_file_is / 'high_scores.json'
+    if data.exists():
+        with open(where_this_file_is / 'high_scores.json', 'r') as source:
+            highscores=json.load(source)
+            print(highscores['high_scores'])
+               
 def quit_game(event=None):
     root.destroy()
 
 
 def new_game(event=None):
     canvas.delete("all")
+    highscore()
```

Comment:
This should be in beginning of program, not in `new_game()`.

Reply:
This was only experimental, I wanted to see if it works.